xls/dslx/frontend/ast.h

```diff
@@ -2194,7 +2193,10 @@ class EnumDef : public AstNode {
   const Span& span() const { return span_; }
   std::optional<Span> GetSpan() const override { return span_; }
   NameDef* name_def() const { return name_def_; }
+
   const std::vector<EnumMember>& values() const { return values_; }
+  std::vector<EnumMember>& values() { return values_; }
```

Comment:
I'd suggest to call these mutable accesses `mutable_values()` and below `mutable_members()` as this sounds like it should be an explicit choice by callers.

Reply:
I followed your suggestion but I think this is not the convention the Google C++ style guide/arbiters recommend for new code (i.e. `mutable_` prefix was recommended in the pre "references are ok" era). Might be good to resolve for future code.

Reply:
We don't general name things with mutable_ (I could only find one instance) and instead just have const and non-const overloads.

Reply:
fair enough. Sorry for the noise (I might've been socialized on earlier more strict code-bases, where it is common practice and strongly enforcuraged).

Reply:
No problem. I do recall some earlier Google style which `mutable` was used. I guess it's still used in the protocol buffer generated C++ api.